src/store/mutations.js

```diff
@@ -89,4 +89,7 @@ export default {
   hideCard(state, name) {
     state.hiddenCards.push(name);
   },
+  setLastRoute(state, payload) {
+    state.lastRoute = payload;
+  },
```

Comment:
As mentioned in the state definition files: I suggest to not add any new features to the Vuex store.